### ISSUE TITLE
Accessibility color issues (1.4.3)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#212]
- [ ] Blue link color fails (#1c84c6) - 4.07 on white background
- [ ] Recommendation tag color & buttons (Agua - #23c6c8) fails on white background - 2.1:1
- [ ] Primary color (#1C84C6) as background with white text
- [ ] Table headers – uses “Grey” or #808080 on top of #eee
- [ ] Non-active tabs - #999 on white
- [ ] Wizard steps- white on light green background
- [ ] Collapsed tile header – teal subheader
